tap-snapshots/package.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Is there an issue for this on the tap side?

Reply:
Hmm not sure, I'll look into it - [my latest commit](https://github.com/googleinterns/presm/pull/13/commits/6896f25b6f62539a07c9bd3df9febec04fd01244) removed this issue by remove the `type`:`module` from the top-level `package.json` but I'll look into this issue further and potentially bring it up to TAP.

Reply:
Found this `package.json` to be unnecessary after latest commits - removed.